test/e2e/specs-gutenberg/wp-calypso-gutenberg-focused-launch-spec.js

```diff
@@ -114,6 +115,41 @@ describe( `[${ host }] Calypso Gutenberg Editor: Focused launch on (${ screenSiz
 			);
 		} );
 
+		step( 'Can select free domain suggestion item', async function () {
+			// Click on the free domain suggestion item
+			const freeDomainButtonSelector = By.css( '.domain-picker__suggestion-item.is-free' );
```

Comment:
Do we want to use the `driverHelper.getElementByText()` helper method here as well (since it resembles the user's behavior better, hence increasing our confidence in our code)?

Reply:
Can't do it here because we'll need to first extract the returned free subdomain suggestion by well... using the same CSS selector... so it's just beating around the bush lol. This is fine for now.

Reply:
Thanks for elaborating.